jupyterhub/values.yaml

```diff
@@ -94,7 +94,7 @@ hub:
     timeoutSeconds: 1
   readinessProbe:
     enabled: true
-    initialDelaySeconds: 0
+    initialDelaySeconds: 60
```

Comment:
@yuvipanda this only influence the Ready state of the hub pod, which only influence if the `hub` k8s Service resources will respond with `Service not available` on traffic, so increasing this is not wanted in my mind. The `livenessProbe` above is already set to 60 seconds, but perhaps we want something longer than 60 seconds?